src/token-group/token.tsx

```diff
@@ -17,17 +18,22 @@ interface TokenProps {
   dismissLabel?: string;
   onDismiss?: () => void;
   disabled?: boolean;
+  loading?: boolean;
   readOnly?: boolean;
   errorText?: React.ReactNode;
   errorIconAriaLabel?: string;
   warningText?: React.ReactNode;
   warningIconAriaLabel?: string;
+  alignment?: TokenGroupProps.Alignment;
+  groupContainsImage?: boolean;
+  type?: 'default' | 'file';
   className?: string;
 }
 
 export function Token({
```

Comment:
Should we introduce a separate FileToken component? I think there are about half of the properties now dedicated to the file token.